grpcurl/plan.sh

```diff
@@ -1,10 +1,28 @@
+gopkg="github.com/fullstorydev/grpcurl"
 pkg_name=grpcurl
 pkg_origin=core
 pkg_description="Like cURL, but for gRPC: Command-line tool for interacting with gRPC servers"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source="https://github.com/fullstorydev/grpcurl/cmd/grpcurl"
-pkg_upstream_url="https://github.com/fullstorydev/grpcurl"
-pkg_version="0.1.0"
-pkg_license=('custom')
+pkg_source="https://$gopkg/cmd/grpcurl"
+pkg_upstream_url="https://$gopkg"
+pkg_version="1.1.0"
+pkg_license=('MIT')
 pkg_bin_dirs=(bin)
 pkg_scaffolding=core/scaffolding-go
+scaffolding_go_base_path=github.com/fullstorydev
+
+do_download() {
+  # `-d`: don't let go build it, we'll have to build this ourselves
+  build_line "go get -d $gopkg"
```

Comment:
Just wondering if all the `build_line` lines are necessary?

Reply:
Nope, carried over from some other plan (dex). I'll ditch them.